docs/topics/concept-getting-started-intro.adoc

```diff
@@ -0,0 +1,53 @@
+
+
+[[launcher-details]]
+.The {launcher} Application
+As a part of {ProductName}, the {launcher} application provides a hassle-free way of creating functional example applications called Missions as well as an easy way to build and deploy those Missions to OpenShift. The {launcher} application is available link:{link-launcher-oso}[directly on {OpenShiftOnline}]. You also have the option to link:{link-launcher-openshift-local-install-guide}#create-launcher-app[install and run it on your {OpenShiftLocal}].
+
+
+.Missions and Boosters
+A Mission implements a link:http://microservices.io/patterns/microservices.html[Microservice pattern] such as:
+
+* Health Check
+* REST or HTTP endpoint 
+* link:https://docs.openshift.com/online/dev_guide/configmaps.html[Externalized Configuration]
+* Circuit Breaker
+
+Missions use these patterns to show how to create the fundamental building blocks of cloud-native applications and services, such as:
+
+* Creating HTTP APIs
+* Integrating your application with a relational database to provide persistent data storage
+* Implementing the health check and circuit-breaker patterns to ensure that your services can withstand a traffic overload and network issues
+* Externalizing the configuration of your applications to make them more secure and easier to scale
+* Securing your applications with {RHSSO} to provide authentication and authorization functionality
+
+Each Mission has an implementation in one or more runtimes. Each specific implementation is known as a booster as is the actual project that contains your code. Any booster can be updated or extended for your own use case. 
+
+[[build-and-deploy-process]]
+.Process for Building and Deploying to OpenShift 
+
+When using {launcher}, you have the option to create and deploy a booster to OpenShift using the _Build and Deploy to OpenShift_ build process, which is based on link:{link-wf-swarm-runtime-guide}#s2i-build-process[the Source-to-Image (S2I) build process]. _Build and Deploy to OpenShift_ configures OpenShift to pull your booster's code from your GitHub repository, build it, and deploy it to OpenShift.
+
+The benefit of this process is that it handles all the configuration, building, and deployment steps needed to get your booster's code running in OpenShift. It also allows you to quickly deploy code updates to help you rapidly iterate and see your changes in OpenShift. 
+
+OpenShift allows different link:https://docs.openshift.com/online/dev_guide/builds/index.html[build strategies] including Source-to-Image and Jenkins Pipeline. The {ProductName} boosters use the link:https://docs.openshift.com/online/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image strategy] for simplicity of use and implementation. Other build strategies, such as the Jenkins Pipeline, have additional complexities such as more servers for doing builds and webhooks to triggering builds.
```

Comment:
You can have a webhook with S2I build as well. Launch actually creates it for our S2I builds.